packages/react-components/src/components/utils/GridLayoutUtils.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BlockProps } from '../GridLayout';
+
+const TARGET_RATIO = 16 / 9;
+const TOLERANCE_RATIO = 8 / 9;
+
+const isRatioBetterThan = (A: number, B: number, targetRatio?: number): boolean => {
+  const _targetRatio = targetRatio ?? TARGET_RATIO;
+  return Math.abs(_targetRatio - A) < Math.abs(_targetRatio - B);
+};
+
+/**
+ * @private
+ */
+export const calculateBlockProps = (n: number, width: number, height: number): BlockProps => {

```

Comment:
Just by looking at this function definition, I can't understand what it does.
What is `n` here for example.
Unless I look at it's implementation.
Can we add some docs and a more descriptive naming for the variables?